src/components/users/Accounts.jsx

```diff
@@ -235,6 +262,69 @@ const CreateAccount = ({
     );
 };
 
+const RootAccount = ({
+    idPrefix,
+    passwordPolicy,
+    setIsRootValid,
+    accounts,
+    setAccounts,
+}) => {
+    const [password, setPassword] = useState(accounts.rootPassword);
+    const [confirmPassword, setConfirmPassword] = useState(accounts.rootConfirmPassword);
+    const [isPasswordValid, setIsPasswordValid] = useState(false);
+    const isRootAccountEnabled = accounts.enableRoot;
+
+    useEffect(() => {
+        setIsRootValid(isPasswordValid || !isRootAccountEnabled);
+    }, [setIsRootValid, isPasswordValid, isRootAccountEnabled]);
+
+    const rootAccountCheckbox = content => (
+        <Checkbox
+          id={idPrefix + "-enable-root-account"}
+          label={_("Enable root account")}
+          isChecked={isRootAccountEnabled}
+          onChange={(_event, enable) => setAccounts(ac => ({ ...ac, enableRoot: enable }))}
+          body={content}
+        />
+    );
+
+    const passphraseForm = (
+        <PasswordFormFields
+          idPrefix={idPrefix}
+          policy={passwordPolicy}
+          password={password}
+          setPassword={setPassword}
+          passwordLabel={_("Passphrase")}
+          confirmPassword={confirmPassword}
+          setConfirmPassword={setConfirmPassword}
+          confirmPasswordLabel={_("Confirm passphrase")}
+          rules={[]}
+          setIsValid={setIsPasswordValid}
+        />
+    );
+
+    useEffect(() => {
+        setAccounts(ac => ({ ...ac, rootPassword: password, rootConfirmPassword: confirmPassword }));
+    }, [setAccounts, password, confirmPassword]);
+
+    return (
+        <Form
+          isHorizontal
+          id={idPrefix}
+        >
+            <Title
+              headingLevel="h2"
+              id={idPrefix + "-title"}
+            >
+                {_("System")}
+            </Title>
+            <Form>
```

Comment:
This <Form> seems to be superfluous..

Reply:
Right, you want one form per page. Remove the <Form> wrapper from the CreateAccount component, and put it in the Accounts component so that it wraps also your newelly added one.

Reply:
Updated